src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('AvailableTasksComponent', () => {
   const mockLocationService = jasmine.createSpyObj('mockLocationService', ['getLocations']);
   const mockLocations: dtos.Location[] = getMockLocations();
   const mockTaskService = jasmine.createSpyObj('mockTaskService', ['searchTask', 'claimTask']);
-  const mockInfoMessageCommService = jasmine.createSpyObj('mockInfoMessageCommService', ['nextMessage']);
+  const mockInfoMessageCommService = jasmine.createSpyObj('mockInfoMessageCommService', ['nextMessage', 'addMessage', 'removeAllMessages']);
```

Comment:
Fixing up some unit tests on the workallocation-3 branch, that are currently failing.